src/darksend.cpp

```diff
@@ -2313,8 +2313,9 @@ void CDarksendPool::UpdatedBlockTip(const CBlockIndex *pindex)
     pCurrentBlockIndex = pindex;
     LogPrint("privatesend", "CDarksendPool::UpdatedBlockTip -- pCurrentBlockIndex->nHeight: %d\n", pCurrentBlockIndex->nHeight);
 
-    if(!fLiteMode && masternodeSync.RequestedMasternodeAssets > MASTERNODE_SYNC_LIST)
+    if(!fLiteMode && masternodeSync.GetAssetID() > MASTERNODE_SYNC_LIST) {
```

Comment:
It seems to me it would be better to call getter methods rather than access object data directly.  For example this would make it much easier to add locking in a well encapsulated way if needed.